.github/workflows/test-ui.yml

```diff
@@ -0,0 +1,51 @@
+name: Test UI
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
```

Comment:
I see a warning on this line. The latest version of this action is `v2.4.1`, should we upgrade it.
And you are using yarn to manage dependencies, should we use `cache: 'yarn'` as well?
And please check all other actions and upgrade them to the latest version as well 🙇